### ISSUE TITLE
Dissapearing Items

### ISSUE THREAD

**Issue by CommonOctopus** [Flanks255/simplybackpacks#24]
**Death removes all items in the backpack**

All of the contents within my backpack up and disappeared on me when I died, needless to say, not a very enjoyable experience.
If it makes a difference, I was using an uncommon backpack when this occurred.

_1.15.2
Forge 31.2.31_

**Comment by Flanks255** [comment]
Generally these problems are caused by another mod messing with the NBT data of the item, i've never been able to reproduce it either way.

**Comment by gloomyMoron** [comment]
> Generally these problems are caused by another mod messing with the NBT data of the item, i've never been able to reproduce it either way.

And I didn't die. I was swimming and exploring a sunken ship and wound up taking a lot of damage due to lack of air. I got to the surface without dying and fought off a Plague Rat (from the Rats mod; it never touched me but it was eating *something* in the water nearby). When I had time to check my backpack (Uncommon tier), it was empty.

MC: 1.15.2
Forge: 31.2.31
SB: 1.15.2-1.4.6

**Comment by Flanks255** [comment]
Interesting... very interesting...

**Comment by gloomyMoron** [comment]
I may have unwittingly dropped and picked it up. I don't think I did but I was panicked and might have hit Q without noticing. So it might be caused by that other issue with Autopick up being on. I've tested that issue since and having Autopickup on, and dropping or moving your bag from an inventory to an inventory makes it so that the next stack you pickup clears the bag.
________
>I just had this same bug happen, but my backpack wasn't dropped. What I think happened was that there was enough space for one or two more of a stack with every other slot filled and when it went to pick up the stack that put it over its capacity limit, it emptied the pack. I *don't think* I dropped it, but I was panicking because I was dying of suffocation (I lived and didn't die but my bag was empty).

>_Originally posted by @gloomyMoron in https://github.com/Flanks255/simplybackpacks/issues/20#issuecomment-663765857_

**Comment by Flanks255** [comment]
That still makes no sense lol

**Comment by Flanks255** [comment]
I "hopefully" fixed this in 1.16, havnt uploaded the version to curse yet, its being tested on forgecraft at the moment, if it seems to fix it, ill apply it to 1.15 as well.

**Comment by shdwdrake8** [comment]
Just want to say I had this issue too. Died with a full Uncommon Backpack with auto-pickup enabled and when I picked up all my gear again (I had more on me because I made some armor and weapons to go after my corpse with) the backpack "picked up" some of my items and deleted everything it originally had in it. My modpack is the current version of Enigmatica 6. This mod is sweet so I hope you get it situated. Thanks.
